realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/ManagementRealmTests.java

```diff
@@ -51,7 +53,42 @@
     @Rule
     public RunInLooperThread looperThread = new RunInLooperThread();
 
-    @Ignore("TODO: Test is currently flaky. See https://github.com/realm/realm-java/pull/4066")
+    // This is primarily a test making sure that an admin user actually connects correctly to ROS.
```

Comment:
are you considering the fact that you can send a Permission & get an error code a "correct connection to ROS"?